### DISCUSSION TITLE
Are the docs wrong or am I wrongly catching the transaction errors?

### DISCUSSION THREAD

**Discussion by AndreGraca3** [prisma/prisma#20805] [Q&A] [answered]
### Bug description

Ive posted everywhere so here it goes:

docs say this:
![image](https://github.com/prisma/prisma/assets/16958881/7cdd0d48-486c-4f9f-b4ea-f23883a05878)

but when i do this it crashes the app:
![image](https://github.com/prisma/prisma/assets/16958881/397aaca4-231e-44c1-a048-5df9d2d40b3c)

### How to reproduce

To reproduce create a transaction with any kind of error. It will not be catched in an outer try catch like docs say.

### Expected behavior

Expected the transaction function to throw an exception and for it to be catched like docs say.

### Prisma information

```prisma
// Add your schema.prisma
```

```ts
// Add your code using Prisma Client
```

### Environment & setup

OS: Windows
Node: 18.7.0

### Prisma Version

```
Prisma 5.1.1
```

**Comment by aqrln** [comment, marked as answer]
You’re missing the `await` before `prismaClient.$transaction` on the screenshot

**Reply by ludralph** [reply]
Hi @AndreGraca3 👋 

do you have any more questions around this issue? If no, it would be really helpful if you could quickly **mark this Discussion as _answered_** to indicate that your issue has been resolved. 

In case you do have any outstanding questions, feel free to let us know and we'd be happy to help! If this discussion is no longer relevant, you can also go ahead and close it.
